src/composites/FeaturesContainer/Overview/OverviewCollector.ts

```diff
@@ -0,0 +1,103 @@
+                    if (factorGroupName === "tqi") { /* Does nothing for now */ }
+                    else if (factorGroupName === "product_factors") {
+                        if (Object.prototype.hasOwnProperty.call(factorGroup, factorName)) {
+                            const factor = factorGroup[factorName];
+
+                            overviewData.qualityFactorNodes.push({
+                                name: factorName,
+                                value: factor.value,
+                                description: factor.description,
+                                eval_strategy: factor.eval_strategy,
+                                normalizer: factor.normalizer,
+                                utility_function: extractUtilityFunctionName(factor.utility_function),
+                            });
+
+                            if (factor.value < 0.2) {overviewData.qualityFactorsCount[0]++;}
+                            else if (factor.value >= 0.2 && factor.value < 0.4) {overviewData.qualityFactorsCount[1]++;}
+                            else if (factor.value >= 0.4 && factor.value < 0.6) {overviewData.qualityFactorsCount[2]++;}
+                            else if (factor.value >= 0.8 && factor.value < 0.8) {overviewData.qualityFactorsCount[3]++;}
+                            else if (factor.value >= 0.8 && factor.value <= 1.0) {overviewData.qualityFactorsCount[4]++;}            
+                        }        
+                    }
+                    else if (factorGroupName === "quality_aspects") {
+                        if (Object.prototype.hasOwnProperty.call(factorGroup, factorName)) {
+                            const aspect = factorGroup[factorName];
+
+                            overviewData.qualityAspectNodes.push({
+                                name: factorName,
+                                value: aspect.value,
+                                description: aspect.description,
+                                eval_strategy: aspect.eval_strategy,
+                                normalizer: aspect.normalizer,
+                                utility_function: extractUtilityFunctionName(aspect.utility_function),
+                            });
+
+                            if (aspect.value < 0.2) {overviewData.qualityAspectsCount[0]++;}
+                            else if (aspect.value >= 0.2 && aspect.value < 0.4) {overviewData.qualityAspectsCount[1]++;}
+                            else if (aspect.value >= 0.4 && aspect.value < 0.6) {overviewData.qualityAspectsCount[2]++;}
+                            else if (aspect.value >= 0.8 && aspect.value < 0.8) {overviewData.qualityAspectsCount[3]++;}
+                            else if (aspect.value >= 0.8 && aspect.value <= 1.0) {overviewData.qualityAspectsCount[4]++;}                          
+                        }
+                    }
+                    else console.log("Error in counter, factor group name not found.");
```

Comment:
Do we need this console log? if this is truly an error case, lets throw an exception here.

Reply:
Console log unnecessary, various logs removed throughout overview functions.